.github/CHANGELOG.md

```diff
@@ -23,6 +23,29 @@ Ashish Panigrahi
 
 <h3>New features since last release</h3>
 
+* Added a sparse hamiltonian observable and the functionality to support computing its expectation
+  value. [(#1398)](https://github.com/PennyLaneAI/pennylane/pull/1398)
+
+  For example, the expectation value of a sparse hamiltonian, an identity matrix in this example,
+  can be computed as:
+
+  ```python
+  dev = qml.device("default.qubit", wires=2)
+
+  @qml.qnode(dev, diff_method="parameter-shift")
+  def circuit(param, H):
+      qml.PauliX(0)
+      qml.SingleExcitation(param, wires = [0, 1])    
+      return qml.expval(qml.SparseHamiltonian(H, wires=[]))
```

Comment:
How come the wires aren't specified here?

Reply:
I corrected that line to `return qml.expval(qml.SparseHamiltonian(H))`. I have set the wires to a default value of the first wire here, so the user does not need to define the wires here. This is because the Hamiltonian matrix is not applied to specific wires and defining wires might be a bit confusing.